packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,11 +101,20 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether the announcement should interrupt any existing announcement,
+  /// or queue after it.
+  ///
+  /// On the web this option uses the aria-live level to set the assertiveness
```

Comment:
I will probably remove anything related to aria-live in the doc string. It is an implementation detail and will probably confuse reader